src/components/entity/FAQ.tsx

```diff
@@ -40,10 +40,10 @@ const FAQ = (props: FAQItem) => {
             size="1.5em"
           />
         </div>
-      </div>
+      </button>
       <div
         ref={faqRef}
-        className="overflow-y-hidden duration-500 h-0 transition-[height]"
+        className="overflow-hidden duration-500 h-0 transition-[height]"
```

Comment:
This also needs to have `visibility: hidden;` when collapsed so that focusable elements inside it cannot be tabbed to. To make sure it doesn't disappear before the transition ends, you can do this: https://jsfiddle.net/425dzcn7/

Reply:
Thanks, updated.